frostsnapp/lib/wallet.dart

```diff
@@ -650,7 +642,20 @@ class WalletDrawer extends StatelessWidget {
                       stream: GlobalStreams.deviceListSubject,
                       builder: (context, snapshot) {
                         final n = snapshot.data?.state.devices.length;
-                        return n == null ? SizedBox.shrink() : Text('$n');
+                        return n == null
+                            ? SizedBox.shrink()
+                            : CircleAvatar(
+                                radius: 12,
+                                backgroundColor: theme.colorScheme.primary,
+                                child: Text(
+                                  '$n',
+                                  style: TextStyle(
+                                    color: theme.colorScheme.onPrimary,
+                                    fontSize: 12,
+                                    fontWeight: FontWeight.w500,
+                                  ),
+                                ),
+                              );
```

Comment:
Navigation drawer badges are not highlighted: https://m3.material.io/components/navigation-drawer/specs

Reply:
Easy feel free to fixup, thanks!